content/ui/popup.js

```diff
@@ -0,0 +1,141 @@
+import { injectServiceWorker } from "../../service_worker/background.js";
+
+const loginForm = document.getElementById("loginForm");
```

Comment:
we should get these element in the `DOMContentLoaded` listener since these elements might not be there when it is first loaded

Reply:
Yes we can but then we will be not able to use some of these elements in the functions outside the **DOMContentLoaded** listener

Reply:
All functions are being called inside the DOMContentLoaded listener only. This approach would not work properly anyway, there's a chance it might break in some cases.

you can keep a global reference to the variable to access in all functions or get the element in the function itself where it is being used